deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/resource/OpenApi.java

```diff
@@ -56,6 +58,13 @@
  */
 @Path("/datasets/{datasetId}/api")
 public class OpenApi {
+	
+	/**
+	 * Name for parameter that allows enabling allowing all origins for CORS. 
+	 */
+	public static final String PARAMETER_CORS_ALLOWALL = "deegree.oaf.openapi.cors.allow_all";
+	
+	private final boolean corsAllowAll = TunableParameter.get(PARAMETER_CORS_ALLOWALL, false);
```

Comment:
I believe here by default tab got added to each line which made the formatting look different in GitHub.

Reply:
Hm, for me it does not look different on GitHub - how does it look different for you?

![image](https://user-images.githubusercontent.com/375475/228486393-4b6c0bc2-c447-48f1-bce3-a2c80d2c1816.png)

Reply:
Strange, to me it looks shifted to right with `private` in L67 starting from the middle of `private` in L69. 
![Screenshot 2023-03-29 at 15 32 52](https://user-images.githubusercontent.com/28185404/228555385-ce6a1bae-b705-449f-a844-cdd6f1179eef.png)